src/library/solrize.py

```diff
@@ -210,7 +213,11 @@ def process_hash_list(document_datasets):
                         sub_list_data[element_name] = fa['@'+element_name]
                         del fa['@'+element_name]
 
-                fa['id'] = utils.get_hash_for_identifier(json.dumps(fa))
+                identifier_indices[hashed_iati_identifier] = identifier_indices.get(hashed_iati_identifier, 0) + 1
```

Comment:
I thought it made more sense since this is a publicly visible field, and for many end-users 1-based would be more intuitive. But can easily make it 0-based when convert to non-draft

Reply:
I think we want to encourage people NOT to use the Id field anyway, as we may need to change scheme later and we should say anything you need is in other exact fields that you don't need to take apart and won't change (and if it's not we'll fix that). So 0 or 1 based, whatever you think.